### DISCUSSION TITLE
Customizable count of icons displayed in the tray

### DISCUSSION THREAD

**Discussion by FoxWMulder** [valinet/ExplorerPatcher#2296] [Ideas]
I am experiencing constant problems with tray icons disappearing. Usually there is no room for more than two icons. Often the program icon in the tray is not displayed for about an system clock and it is not in the drop-down list either. System settings do not help, nor does dragging icons manually. I would like to 1) customize the number of icons displayed in tray 2) be able to pin the icon so that it is not hidden.

**Comment by pyrates999** [comment]
I would set it to show all system tray icons instead.  But when only showing only active system tray icons, the system tries to be smart and show the ones it thinks you need.

**Comment by onilink146** [comment]
I am also experiencing this issue. I am not sure how long it has been going on for after recently needing to show an icon for Autohotkey. My taskbar always shows Windows Defender, Steam, and Teamspeak 3 icon (other than volume and internet icons). I went to the taskbar settings (windows) and "Other system tray icons" option was missing. I had to reinstall the program for the option to show again. After getting the option back I tried to manually enable tray icons to show. The icon is set on but doesn't enable while ExplorePatcher is installed.
